### PR TITLE
added supersonic infow boundary condition support

### PR DESCRIPTION
**Pull request by johnmoore4**

### CODE REVIEW

File: flow360/component/flow360_params/flow360_params.py [hunk 1]
```diff
@@ -131,6 +131,13 @@ class SubsonicInflow(Boundary):
     totalTemperatureRatio: PositiveFloat
     rampSteps: PositiveInt
 
+class SupersonicInflow(Boundary):
+    """SupersonicInflow boundary"""
+
+    type = pd.Field("SupersonicInflow", const=True)
+    density: PositiveFloat = pd.Field(alias="Density")
+    velocity: Tuple[float, float, float] = pd.Field(alias="Velocity")
+    pressure: PositiveFloat = pd.Field(alias="Pressure")
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:140]: 1. Are all parameters required, or some of them are optional?
2. Can you write docstring to it following numpy style:
see this for example: https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/flow360/component/flow360_params/flow360_params.py#L191

3. Can you add test for this BC, somewhere here for example:
https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/tests/test_flow360_params.py#L89
4. Add this BC to BC types list:
https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/flow360/component/flow360_params/flow360_params.py#L174

and update docstring for boundaries: https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/flow360/component/flow360_params/flow360_params.py#L539

Comment (johnmoore4) [reply flow360/component/flow360_params/flow360_params.py:140]: All parameters are required

File: flow360/component/flow360_params/flow360_params.py [hunk 2]
```diff
@@ -132,6 +133,36 @@ class SubsonicInflow(Boundary):
...
+    Parameters
+    ----------
+    Density : PositiveFloat
+        Fluid density (non-dimensional). Default freestream fluid density in Flow360 is 1.0.
+
+    Velocity: Tuple[float, float, float]
+        Fluid velocity (non-dimensional), normalized to Mach number.
+    Pressure: PositiveFloat
+        Fluid pressure (non-dimensional). Default freestream fluid pressure in Flow360 = 1.0/gamma = 1.0/1.4.
+                                                                circumferentialForcePerArea[SI=N/m2]/rho_inf/C_inf^2
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:148]: this line should be removed (circumferentialForcePerArea)

Comment (johnmoore4) [reply flow360/component/flow360_params/flow360_params.py]: removed

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**